core/src/main/scala/kafka/raft/RaftManager.scala

```diff
@@ -118,9 +119,20 @@ class KafkaRaftManager[T](
   private val raftIoThread = new RaftIoThread(raftClient)
 
   def startup(): Unit = {
+    // Update the voter endpoints (if valid) with what's in RaftConfig
+    val voterAddresses: util.Map[Integer, AddressSpec] = raftConfig.quorumVoterConnections
+    for (voterAddressEntry <- voterAddresses.entrySet.asScala) {
+      voterAddressEntry.getValue match {
+        case spec: InetAddressSpec => {
+          netChannel.updateEndpoint(voterAddressEntry.getKey, spec)
+        }
+        case invalid: AddressSpec => {
+          logger.warn(s"Skipping channel update for destination ID: ${voterAddressEntry.getKey} " +
```

Comment:
This could be `info` I think in the case of `UnknownAddressSpec`. It is expected behavior to skip the update. We could add a third case for unexpected `AddressSpec` types.

Reply:
Ack.